dashboard/README.md

```diff
@@ -0,0 +1,33 @@
+# Daft Dashboard
+
+## Installation
+
+You will need the following installed prior to running this dashboard fully-featured.
+
+- [Bun](https://bun.sh): `curl -fsSL https://bun.sh/install | bash`
+- [Ray](https://www.ray.io): `uv v; source .venv/bin/activate; uv pip install ray[default]`
+- [Prometheus](https://prometheus.io): `brew install prometheus`
+- [Grafana](https://grafana.com): `brew install grafana`
```

Comment:
Is this all necessary right now given that we just have a blank page still?

Reply:
Not currently, but I will remove them incrementally.

Reply:
Ok, would prefer if we can *add* them incrementally instead :)